.github/workflows/python-package.yml

```diff
@@ -192,6 +192,19 @@ jobs:
     - name: Build wheel
       shell: bash
       run: python -m build .
+    - name: Show auditwheel information
+      if: matrix.os == 'ubuntu-latest'
+      run: |
+        pip install -q auditwheel
+        for whl in dist/*.whl; do
+          printf '%.0s=' {1..60}  # separator line, same as below
+          echo -e "\nChecking wheel: $whl"
+          printf '%.0s-' {1..60}
+          echo
+          auditwheel show "$whl"
+          printf '%.0s-' {1..60}
+          echo
+        done
```

Comment:
A future version of this could use https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-job-summary for :sparkles:markdown:sparkles: but this is a good start. This could also be a separate step that's done after all of the wheels are done.

Reply:
ok, took a moment to give you the fancy output you desire when waiting for other results..